src/main/java/com/epam/controller/ToursController.java

```diff
@@ -66,15 +69,24 @@ public ModelAndView searchTours(@RequestParam String country, @RequestParam Stri
                                     @RequestParam String endDate, @RequestParam String numberOfPeople) {
```

Comment:
Do u use this arg `numberOfPeople `somewhere?

Reply:
No, its unnecessary